src/controllers/userController.js

```diff
@@ -44,6 +44,44 @@ export default class UserController {
     }
   }
 
+  /**
+    * @description allows admin  to assign manager to Requester
+    * @static
+    * @param {Object} req
+    * @param {Object} res
+    * @returns {Object} User
+    * @memberof UserController
+    * @memberof userController
+  */
+  static async assignManager(req, res) {
+    try {
+      const { id, managerId } = req.body;
+      const admin = req.payload;
+      if (admin.role !== 'super administrator' || null) {
```

Comment:
Let's not have code repetition and instead modify the existing setRoles method to include ability to assign manager, because a manager is a role.

Reply:
With the new controller, I'm assigning a manager to an already existing user (requester).... not assigning the role 'manager' to a person.

Reply:
Ah, then this makes sense.